GOVERNANCE.md

```diff
@@ -0,0 +1,45 @@
+# Node.js Web Team Governance
+
+The Node.js Web Team (@nodejs/web) is a team in the Node.js Project that is composed by a set of subteams. Each containing specific responsibilities and goals.
+
+### Node.js Website Team (`@nodejs/nodejs-website`)
+
+The Node.js Website Team is responsible for the technical development of the Node.js Website. This includes the development of the features of the Website, providing stability and security to the Website, and the maintenance of the Website.
```

Comment:
My read of this is that it overlaps with the Web infra team? In my mind a having one team responsible for the content of the website and one for the infra makes sense. Is that what you were trying to describe or something else?

Reply:
Hmmm, the web-infra team was more responsible for the infrastructure of the Website (like the framework, and the technology behind it) whereas the nodejs-website is responsible for writing the ui components, and features of the website.

So neither hold or govern the content itself (if you check the content_vs_code.md) file.

But maybe my explanations on the distinction of these two teams was not good enough?

Reply:
Ah also web-infra would take care of Vercel, CI/CD and tooling

Reply:
I'm wondering if its being sliced into a few too many subteams at this point.  But since I'm not too active on the website side I'll leave it for those who are to decide.

The main discussion after the teams are defined will be the web-infra team and how things are teased appart from the pieces managed by the build WG. I think it will also be a good time to figure out what parts the Foundation infra teams will be able to own as well.

Reply:
What do you believe it's being sliced too much?

The main reasoning here between web-infra and nodejs-website is to lower the entry for contributors on the org and to separate the technical ownership of the website vs the contributors that make the day to day changes.

This is something similar as node/build and node/build-infra

Does that make sense?

Reply:
If I may, I think I'm probably a good example of this to an extent -- I'm absolutely not involved in day-to-day development of the site, but do very much want to be involved in the infra side of things when it comes to the delivery of the site via Vercel and Cloudflare, downloads via Cloudflare R2/NGINX, etc.

Reply:
Also regarding web infra, I think we definitely could sync with the Build WG and the Foundation during Collab Summit and the next Social team meeting.

I also believe that it's completely fine for web-infra to only own Vercel and maybe read access on Cloudflare or specific permissions on Cloudflare for Vercel related stuff 👀

Reply:
cc @nodejs/build-infra

Reply:
node/build and node/build-infra are subteams where everbody who is in build-infra are also in build which I think is a bit different than what's being described.

I don't want to say much more on the slicing since I don't have strong feelings, I just started to wonder if we are going to end up with 5 teams with the same 2 people on them.

Reply:
Hmm not really. nodejs-website will naturally will have way more, also not necessarily people on web-infra will be on nodejs-website.

@MattIPv4 I feel that my descriptions of each team aren't good enough. Would you have some bandwidth to better them based on what we're talking here? 🙈

I also wonder if in a "governance" language if anything is missing here.

I also do wonder if the nodejs.org website is the best place to set the governance charter for the web team... As it does not have a deeicated repository

Reply:
I had felt the descriptions here were okay, but sure, I can take another read through later and see if anything can be reworded to make this distinction clearer 👍 

I think this repo is probably best, I think it's relatiavely obvious that it is the canonical repo for the web team, given the web team looks after the website and this repo is the website.

Reply:
```suggestion
The Node.js Website Team is responsible for the day-to-day technical development of the Node.js Website. This is primarily the development of the website itself, adding new features, pages and components, but also fixing any security issues in the website code, handling operational maintenance, and so on.
```
How about this?